forms-flow-service/src/helpers/helperServices.ts

```diff
@@ -0,0 +1,43 @@
+import moment from 'moment'
+
+class HelperServices {
+
+  public static getLocalDateAndTime(
+    date: string,
+    dateFormat: string,
+    timeFormat: string,
+  ): any {
+    if (!date) {
+      return '-'
+    }
+    return moment(
```

Comment:
Do we require moment here?? I think we can handle it without moment library

Reply:
.format function is available in moment and also there are other features are available in moment so in future also may be we can use it